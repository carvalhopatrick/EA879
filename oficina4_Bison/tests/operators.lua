print(2+4, 2*4, 2-4, 2/4, 2%4, 2^4)
print(2 .. 4)
print(#'this string')
print(2 == 4, 2 ~= 4, 2 <= 4, 2 >= 4, 2 < 4, 2 > 4)
value = 2 == 4 or 2 ~= 4
print(value)
print(2+3+4, 2*3*4, 2-3-4, 2/3/4, 8%5%2, 2^3^4)
print(2+3*4^5, 2^3*4+5, (2+3)*4^5, 2^3*(4+5))
print(2 > 3 and 3 > 4 or 4 < 5)
print(-2^2, -2^3, -2^4, -2^5)
print(-4-2, -4+2)
