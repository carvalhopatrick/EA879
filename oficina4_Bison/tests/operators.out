6	8	-2	0.5	2	16
24
11
false	true	true	false	true	false
true
9	24	-5	0.16666666666667	1	2.4178516392293e+24
3074	37	5120	72
true
-4	-8	-16	-32
-6	-2
